Examples/Swift/ViewController.swift

```diff
@@ -120,8 +118,14 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        // Reset the navigation styling to the defaults
-        DayStyle().apply()
+        //Reload the mapView.
+        setupMapView()
+
+        // Reset the navigation styling to the defaults if we are returning from a presentation.
+        if (presentedViewController != nil) {
+            DayStyle().apply()
```

Comment:
I believe the right way to reset styling to the defaults is use a `styleManager` and probably call `timeOfDayChanged()`... right?

Reply:
This approach won't work because the `ViewController` will always show a `DayStyle` theme, regardless of the time-of-day.

In reality, the call isn't strictly necessary because the only thing the `DayStyle` does that affects the `ViewController` is that it changes the style of the status bar, a thing we could do directly. But I digress.